flink-runtime-web/src/test/java/org/apache/flink/runtime/rest/compatibility/CompatibilityRoutines.java

```diff
@@ -173,6 +194,13 @@ private static void assertCompatible(
                                     "Previously optional query parameter %s is now mandatory.",
                                     oldParam.key));
                 }
+
+                if (!newParam.description.equals(oldParam.description)) {
```

Comment:
these routines are not used by the completeness test, are they? Why do we have to extend them?